### PR TITLE
JBPM-7200: Master details: Refresh button shouldn't change the active…

### PR DESCRIPTION
**Pull request by nmirasch**
… tab

### CODE REVIEW

**Comment by nmirasch** [comment]
Jenkins execute full downstream build

**Comment by nmirasch** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetailsmulti/TaskDetailsMultiPresenter.java [hunk 1]
```diff
@@ -135,6 +135,8 @@ public void setIsForLog(boolean isForLog) {
     }
 
     public void onTaskSelectionEvent(@Observes final TaskSelectionEvent event) {
+        boolean resetTabs = (taskId != event.getTaskId());
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetailsmulti/TaskDetailsMultiPresenter.java]: @nmirasch this comparison seems to break findbugs, one of the reasons is that comparing with a _!=_ could throw a NPE.

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac rebuilt the failing condition, could you take a look? Thanks!

**Comment by nmirasch** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java [hunk 2]
```diff
@@ -164,7 +164,7 @@ public void setIsForLog(boolean isForLog) {
     }
 
     public void onProcessSelectionEvent(@Observes ProcessInstanceSelectionEvent event) {
-
+        boolean resetTabs = (processInstanceId != event.getProcessInstanceId());
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java]: @nmirasch this one also needs to be fixed.

**Comment by nmirasch** [comment]
@cristianonicolai done, thanks!

**Pull request merged**